util/design/get-lfsr-coeffs.py

```diff
@@ -119,14 +119,15 @@ def main():
         if args.fib:
             lfsrType = 'FIB_XNOR'
 
-            wget.download(FIB_URL)
-            cmd = ['pdftotext %s.pdf' % PDF_NAME, '> %s.txt' % PDF_NAME]
+            if not os.path.isfile(args.fib):
+                print("File %s does not exist" % args.fib)
+                sys.exit(1)
```

Comment:
When testing, this error kept popping up. To fix it I had to replace `args.fib` with something like `ABS_PATH + '\' + args.fib` where `ABS_PATH` was computed as `os.getcwd()` before the `if not os.path.exists(args.tempfolder)`  branch.

I think that maybe `os.chdir(args.tempfolder)` in line 117 was causing this.

Anyways, it all works well after resolving the path.

Reply:
Interesting... that might be OS dependent.
I changed this so that the `args.fib` is converted to an absolute path first.
Can you try again?

Reply:
I've just tried it. It works perfectly now.